app/views/poems/show.html.erb

```diff
@@ -7,6 +7,13 @@
   </p>
 </div>
 <div>
+  <%= link_to "読んだよ", read_poems_path(@poem), method: :post, action: :create, class: "btn btn-primary"%>
+  <% @read_poems.each do |read_poem| %>
+    <span><%= link_to image_tag(read_poem.user.image_url, class: "img-circle", width: 35 ),
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)